ammico/expressions.py

```diff
@@ -0,0 +1,108 @@
+import numpy as np
+import webcolors
+import pandas as pd
+from collections import defaultdict
+
+import colorgram
+
+import colour
+
+from ammico.utils import get_color_table
+
+
+def rgb2name(c, merge_color=True, delta_e_method="CIE 1976"):
+    """This function takes a rgb color as input and returns the closest color name from the CSS3 color list.
```

Comment:
I updated the docstrings. Please always use google style docstrings and also consult the [PEP standard for docstrings](https://peps.python.org/pep-0257/).